_posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.md

```diff
@@ -94,4 +94,53 @@ You can also view our ["Hacking Bureaucracy"
 presentation](https://speakerdeck.com/18f/hacking-bureaucracy-18f-demo-day-9-may-2014)
 from 18F Demo Day on May 9th:
 
-{% oembed https://speakerdeck.com/18f/hacking-bureaucracy-18f-demo-day-9-may-2014 %}
```

Comment:
For this one, it's using an `oembed` plugin to build the iframe. Rather than try to figure out how to configure the plugin or anything, I instead replaced the `oembed` call with the output that it rendered. It's a hack but it works! Then I was able to set the `title` attribute.